src/query/expression/src/function.rs

```diff
@@ -98,7 +98,8 @@ pub enum FunctionEval {
 #[derive(Clone)]
 pub struct FunctionContext {
     pub tz: TzLUT,
```

Comment:
Can we remove it?

Reply:
Cannot. Date_format need this. Jeff not support format string with no ascii format

Reply:
> Can we remove it? Or convert from jiff_tz?

remove.